src/screens/AddSiteFormScreen.js

```diff
@@ -397,6 +394,9 @@ const styles = {
         paddingRight: 20,
         flexDirection: 'row',
         justifyContent: 'space-between'
+    },
+    addSiteCheckboxRowStyle: {
+        margin: 0
```

Comment:
Started adding some basic styling to the checkboxes. They look better with no margin, but still can be improved.